docs/getting_started/typescript/hello_world_in_typescript/index.md

```diff
@@ -88,142 +158,155 @@ Create the `package.json` file in the root of your project and add the following
 
-:::tip
-No matter how many inputs you pass to a Workflow, it's a best practice to use an object. There are some JSON restrictions that you should be aware of, and you can learn more about the kinds of inputs you can pass to a Workflow in the [Workflow parameters](https://docs.temporal.io/dev-guide/typescript/foundations#workflow-parameters) section of the Temporal documentation.
+You've created your two Activities. Now you'll coordinate them using a Temporal Workflow.
 
-:::
+## Control application logic with a Workflow
 
-Next, you'll define the Activity that your Workflow will execute.
+Workflows are where you configure and organize the execution of Activities. You define a Workflow by writing a *Workflow Definition* using one of the Temporal SDKs. Review the [Develop Workflows](https://docs.temporal.io/develop/typescript/core-application#develop-workflows) section of the Temporal documentation for more about Workflows in TypeScript.
 
-## Create an Activity
+Open the file `src/workflows.src` and remove the code in the file since you'll add your own. Then add the following code to import the Activities and configure how the Workflow should handle failures:
 
-You use Activities in your Temporal Applications to execute non-deterministic code or perform operations that may fail.
+```typescript
+import * as workflow from '@temporalio/workflow';
+
+// Only import the activity types
+import type * as activities from './activities';
 
-For this tutorial, your Activity won't be complex; you'll create an Activity that takes a string as input and uses it to create a new string as output, which is then returned to the Workflow. This will let you see how Workflows and Activities work together without building something complicated.
 
-With the Temporal TypeScript SDK, you define Activities similarly to how you define Workflows: using an exportable TypeScript module.
+// Load Activities and assign the Retry Policy
+const { getIP, getLocationInfo} = workflow.proxyActivities<typeof activities>({
+  retry: {
+    initialInterval: '1 second',
+    maximumInterval: '1 minute',
+    backoffCoefficient: 2,
+    // maximumAttempts: 5,
+  },
+  startToCloseTimeout: '1 minute',
+});
+```
+
+The Temporal TypeScript SDK requires that Workflows and Activities run in separate environments. Temporal Workflows must be deterministic, and so Workflows are run in a sandbox that checks code for determinism. Since you run your non-deterministic operations in Activities, you configure your Workflow to call Activities through a proxy. That's why you import their types rather than the functions themselves.
```

Comment:
Maybe link to the page about deterministic constraints because we aren't explaining here WHY workflows must be deterministic

https://docs.temporal.io/workflows#deterministic-constraints